File: sdk/async-scala/src/main/scala/io/swagger/client/model/BusStopList.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class BusStopList (
  errorcode: Option[String],
errormessage: Option[String],
numberofresults: Option[Integer],
timestamp: Option[String],
results: Option[List[Stop]]
)

File: sdk/javascript-closure-angular/API/Client/BusStopList.js
goog.provide('API.Client.BusStopList');

/**
 * @record
 */
API.Client.BusStopList = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.BusStopList.prototype.errorcode;

/**
 * @type {!string}
 * @export
 */
API.Client.BusStopList.prototype.errormessage;

/**
 * @type {!number}
 * @export
 */
API.Client.BusStopList.prototype.numberofresults;

/**
 * @type {!string}
 * @export
 */
API.Client.BusStopList.prototype.timestamp;

/**
 * @type {!Array<!API.Client.Stop>}
 * @export
 */
API.Client.BusStopList.prototype.results;

